examples/ckbtc/wallet/backend/index.ts

```diff
@@ -29,66 +30,177 @@ export default Canister({
     init: init([], setupCanisters),
     postUpgrade: postUpgrade([], setupCanisters),
     getBalance: update([], nat64, async () => {
-        return await ic.call(ckBTC.icrc1_balance_of, {
-            args: [
+        if (process.env.AZLE_TEST_FETCH === 'true') {
+            const response = await fetch(
+                `icp://${(ckBTC as any).principal.toText()}/icrc1_balance_of`,
```

Comment:
Get the ckBTC and minter principals from the environment variables, use the functions like we do elsewhere